code/client/clrcore/External/Blip.cs

```diff
@@ -482,6 +482,34 @@ public void RemoveNumberLabel()
 			API.HideNumberOnBlip(Handle);
 		}
 
+		/// <summary>
+		/// Adds a cone for this <see cref="Blip"/>
+		/// </summary>
+		/// <param name="width">Width of the cone (0.0f - 3.0f)</param>
+		/// <param name="length">Length of the cone</param>
+		/// <param name="heading">Heading of the cone in radians (degrees * pi / 180)</param>
+		/// <param name="color">Color of the cone (see <see href="https://docs.fivem.net/docs/game-references/hud-colors/">hud colors</see>/></param>
+		/// <param name="unk0">Unknown at this time</param>
+		/// <param name="unk1">Unknown at this time</param>
+		/// <param name="unk2">Unknown at this time</param>
+		/// <param name="unk3">Unknown at this time</param>
+		public void AddCone(float width, float length, float heading, int color, float unk0 = -1.0f, float unk1 = 1.0f, float unk2 = 1.0f, int unk3 = 1)
```

Comment:
1. `ShowCone` and `HideCone` would be better, "add" implies we can have multiple.
2. It seems like `width` is the angle in radians, but used as: `angle * 2`, i.e.: ~3.14 is a full circle (oddly...).
3. `distance` would be more descriptive than `length`.